src/user-guide/removing-workflows.rst

```diff
@@ -110,12 +104,13 @@ A note on symlink directories
    :cylc:conf:`global.cylc[install][symlink dirs]`, you might want to read
    this explanation of how Cylc handles them during cleaning.
 
-If you manually delete a run directory (e.g., using ``rm`` or the file
-manager), only the symlinks themselves will be deleted, not the actual targets.
-In contrast, ``cylc clean`` follows the symlinks and deletes the targets.
+If you manually delete a run directory (e.g., using the ``rm`` command or the
+file manager), only the symlinks themselves will be deleted, not the actual
+targets. In contrast, ``cylc clean`` follows the symlinks and deletes the
+targets.
 
-- It does this for the symlinks that can be set in
-  :cylc:conf:`global.cylc[install][symlink dirs]` only, not any custom
+- It does this only for the symlinks that can be set in
```

Comment:
Take or leave, I found myself re-reading this sentence, although perhaps personal preference.
```suggestion
- It only does this for the symlinks that are set in
```